.github/workflows/cuda.yml

```diff
@@ -84,19 +82,20 @@ jobs:
             compiler: gcc
             python_version: "3.10"
             cuda_version: "11.8.0"
+            linux_version: "ubuntu20.04"
             image: nvcr.io/nvidia/cuda:11.8.0-devel-ubuntu20.04
```

Comment:
Remove the `image` here?

Reply:
ah yep, missed this one. Thank you!

Removed in https://github.com/microsoft/LightGBM/pull/6463/commits/e8b679ab880f802feede466d48da7702f84a751c